synapse/lib/stormhttp.py

```diff
@@ -567,13 +566,13 @@ async def _httpRespJson(self, encoding=None, errors='surrogatepass'):
             else:
                 encoding = await s_stormtypes.tostr(encoding)
 
-            return json.loads(valu.decode(encoding, errors))
+            return s_json.loads(valu.decode(encoding, errors))
```

Comment:
Discuss + related:

`json.detect_encoding()` (used a few lines above ) has been unchanged for 9 years ( https://github.com/python/cpython/blame/3.13/Lib/json/__init__.py#L244-L271 ). I wonder if we could vendor this? Then we could drop the stdlib import for this file?